src/search/search-index/add.js

```diff
@@ -225,3 +225,40 @@ async function performIndexing(indexDoc) {
         console.error(err)
     }
 }
+
+/**
+ * Adds a reverse index doc (bookmarks or visits) entry.
```

Comment:
The JSDoc looks great! Thanks for doing that 😃 This line should be something like "Adds a meta (bookmark or visit) entry into specified reverse index doc" though, as it doesn't add a new reverse index doc